frontend/integration-tests/protractor.conf.ts

```diff
@@ -179,6 +183,18 @@ export const config = {
         password_manager_enabled: false,
       },
     },
+    'moz:firefoxOptions': {
+      binary: '/usr/bin/firefox',
+      args: [
+        ...(process.env.NO_HEADLESS ? [] : ['--headless']),
+        '--safe-mode',
+        '--width=1920',
+        '--height=1200',
+        '--MOZ_LOG=timestamp,nsHttp:0,sync',
+        `--MOZ_LOG_FILE=${screenshotsDir}/browser`,
```

Comment:
```suggestion
        `--MOZ_LOG_FILE=${screenshotsDir}/browser.log`,
```

Reply:
That would end up creating `browser.log.log`. Do you want that?

Reply:
> That would end up creating `browser.log.log`. Do you want that?

That's surprising. OK, disregard :)